ReaderClass/TemporaryReader.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ReaderClass
+{
+    public class TemporaryReader : Reader
+    {
+        public DateTime EndOfAccess { get; set; }
+
+        public string[] AccessableDepats = { };
```

Comment:
Лучше назвать свойство
```suggestion
        public string[] AccessableDepartments = { };
```
 или
```suggestion
        public string[] AccessableDepts = { };
```